src/pybind/mgr/dashboard/controllers/auth.py

```diff
@@ -95,19 +95,25 @@ def create(self, username, password, ttl: Optional[int] = None):
                 try:
                     if fsid in multicluster_config['config']:
                         existing_entries = multicluster_config['config'][fsid]
-                        if not any((entry['user'] == username or entry['cluster_alias'] == 'local-cluster') for entry in existing_entries):  # noqa E501 #pylint: disable=line-too-long
+                        for entry in existing_entries:
```

Comment:
better name than entry and existing entries?

Reply:
Updated